packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -238,6 +240,17 @@ export class LitDevSearch extends LitElement {
 
   private _listboxClicked = false;
 
+  update(changed: PropertyValues<this>) {
+    if (!this._searchText && this._inputEl) {
+      console.log(this._inputEl.value);
+      // this is required on hydration to make sure we don't blow away the
+      // input text due to the `live` directive which is necessary to prevent
+      // issues with typing on the virtual keyboard in Safari on iOS.
```

Comment:
https://github.com/material-components/material-web/pull/877

If you type and set `this._searchText` Lit only knows that the binding at `.value` is stale. `live` prevents this from happening because it first checks the value of the input before setting it.